examples/cms-agilitycms/.gitignore

```diff
@@ -0,0 +1,2 @@
+.env*.local
```

Comment:
I think we can remove this line, as it’ll be injected by `create-next-app`?

Reply:
Better to keep it until we remove the manual download from all examples